tests/test_icons_products_to_deploy_QCI96.py

```diff
@@ -0,0 +1,17 @@
+# https://polarion.engineering.redhat.com/polarion/redirect/project/QuickstartCloudInstaller/workitem?id=QCI-96
+
+# In the "Select products to deploy" screen, when you hover over the "i" icon by each product, text about the product is displayed. This is true whether you have the product selected or not.
```

Comment:
I know we don't have a standard line length but for comments I would probably keep them around 80 chars, rather than have just one long line of text.